src/main/docker/Dockerfile

```diff
@@ -1,10 +1,27 @@
 FROM eclipse-temurin:17.0.7_7-jdk
 
-ADD kafdrop.sh /
-ADD kafdrop*tar.gz /
+COPY kafdrop*.tar.gz /tmp/kafdrop.tar.gz
 
-RUN chmod +x /kafdrop.sh
+RUN mkdir /opt/kafdrop && \
+    tar \
```

Comment:
This became a lot more complex than it used to be. Let's simplify this.
* Through `ADD`, Docker can extract the zip to the `/opt/kafdrop`. That causes a bit of an ugly directory structure, but nobody needs to go through it, so it doesn't matter
* The `chmod` can be prevented by adding the execution permission in Git. I created #539 for that

With these changes, the `Dockerfile` can be come this simple:
```
FROM eclipse-temurin:17.0.7_7-jdk

WORKDIR /opt/kafdrop
ADD kafdrop*tar.gz .
COPY kafdrop.sh .

RUN groupadd -g 1000 kafdrop && \
    useradd -u 1000 -g kafdrop kafdrop && \
    chown --recursive kafdrop:kafdrop /opt/kafdrop

USER kafdrop

ENV PATH="/opt/java/openjdk/bin:$PATH"

EXPOSE 9000

ENTRYPOINT ["/opt/kafdrop/kafdrop.sh"]
```